pkg/scheduler/util/utils_test.go

```diff
@@ -257,3 +266,36 @@ func TestPatchPodStatus(t *testing.T) {
 		})
 	}
 }
+
+func TestGetPodStartTime(t *testing.T) {
```

Comment:
IMO, we can cover this in `TestGetEarliestPodStartTime`, and make `GetPodStartTime` acts as a private method `getPodStartTime`. For testccases we covered here should also be tested in `TestGetEarliestPodStartTime`. If we do as what I advised, we can reduce the repetitive tests.

Reply:
Thanks for your suggestion, I've moved the GetPodStartTime() test to TestGetEarliestPodStartTime() to reduce duplication of testing. Thanks againThanks for your suggestion, I've moved the GetPodStartTime() test to TestGetEarliestPodStartTime() to reduce duplication of testing. Thanks again